astropy/time/formats.py

```diff
@@ -690,10 +690,19 @@ def _default_scale(self):
 
 class TimeUnix(TimeFromEpoch):
     """
-    Unix time: seconds from 1970-01-01 00:00:00 UTC, ignoring leap seconds.
+    Unix time (UTC): seconds from 1970-01-01 00:00:00 UTC, subject to leap seconds.
```

Comment:
The words `subject to` are ambiguous. Please leave this as it was originally.

Reply:
Ignoring is just wrong. UTC is the only timescale that has leap seconds. Subject to may be ambiguous, but it is rightly so because it confers to the reader that it is not as simple as "ignoring" and that needs to understand what is actually happening.

Reply:
In this context "ignoring leap seconds" is equivalent to the definition of unix time which is computing the seconds since the epoch assuming exactly 86400 seconds in every day. The actual unix time on a computer *is* as simple as ignoring, no?


The value of Unix time on a computer is related to the UTC time name components (year, yday, hour, min, sec) according to the expression:
```
tm_sec + tm_min*60 + tm_hour*3600 + tm_yday*86 400 + (tm_year-70)*31 536 000 + ((tm_year-69)/4*86 400
```